### ISSUE TITLE
coordinates: repr of SkyCoord for a single coordinate

### ISSUE THREAD

**Issue by adrn** [astropy/astropy#3298]
tl;dr I think we should change the way a single coordinate is displayed via `repr()` to match the format of the multiple coordinate case.

I often use coordinates to quickly transform a pair or a few pairs of coordinates from ICRS <--> Galactic so I can then copy/paste the pair into some web form that is expecting, e.g., a pair of coordinates specified as `192.13814 11.5142`. For a single coordinate, the representation splits the two components, for example:

    >>> coord.SkyCoord(ra=13.5135*u.hourangle, dec=-10.4124*u.degree)
    <SkyCoord (ICRS): ra=202.7025 deg, dec=-10.4124 deg>

making copy-pasting soooooo laborious. For multiple pairs of coordinates, they are listed as RA,Dec pairs:

    >>> coord.SkyCoord(ra=[13.5135,2.3]*u.hourangle, dec=[-10.4124,16.23523]*u.degree)
    <SkyCoord (ICRS): (ra, dec) in deg
        [(202.7025, -10.4124), (34.49999999999999, 16.23523)]>

I propose that we change the single-coordinate case to be like the multiple coordinate case, e.g.:

    <SkyCoord (ICRS): (ra,dec) in deg (202.7025, -10.4124)>

or some other similar form.

Bringing this up for discussion before I start a PR...

cc @eteq

**Comment by andycasey** [comment]
:+1: pls

**Comment by mhvk** [comment]
Funny, we special-cased the single-coordinate case since we felt it looked nicer the way it is now... (see `coordinates.representation.BaseRepresentation.__repr__`). But no objection to changing it now that we've had some real experience!

**Comment by taldcroft** [comment]
:+1:

**Comment by embray** [comment]
I do think the single coordinate syntax looks nicer the way it is now.  But I can also see how the proposed change is more useful in practice, so :+1:

**Comment by adrn** [comment]
@embray I agree it looks nicer, but I lika the easy copy pasta :spaghetti:

**Comment by eteq** [comment]
I'm with @embray that the old way looks slightly nicer... but I see @adrn's point and this approach is *also* better for the case where there's frame attributes (as this will make sure they're kept separate from the data).  So I'm :+1: to this suggestion.
